revolv/static/dashboard.scss

```diff
@@ -399,6 +399,87 @@ $sidebar-icon-gray: #999;
         h6 {
             font-size: 10px;
         }
+
+        .repayment-progress-container {
+            height: 260px;
+            .statistics-graphics-container {
+                height: 260px;
+                width: 100%;
+                position: absolute;
+
+                .svg-graphics-container {
+                // this contains the formatting for the white outside circle - this works
+                    position: absolute;
+                    height: 100%;
+                    width: 100%;
+                }
+
+                .internal-graphics-container {
+                    position: absolute;
+                    left: 50%;
+                    top: 50%;
+                    transform: translate(-50%, -50%);
```

Comment:
Not fully sure how to do a mixin for scss. And also, a mixin for the circle on the project page and the one in the dashboard?